code/controllers/stonedmc/subsystem/job.dm

```diff
@@ -213,7 +212,7 @@ SUBSYSTEM_DEF(job)
 					// If the job isn't filled
 					if((job.current_positions < job.spawn_positions) || job.spawn_positions == -1)
 						JobDebug("DO pass, Trying to assign Player: [player], Level:[level], Job:[job.title]")
-						if(AssignRole(player, job.title))
+						if(AssignRole(player, job))
```

Comment:
Will make everyone share the same `datum/job` (the one in `all_jobs` list). Is this ok ?

Reply:
Not really that different from current behavior that's just one abstraction away from sharing the same ref.